README.md

```diff
@@ -235,31 +235,18 @@ We used the v1 environments.
 
 ### MiniGrid Envs
 
-See https://github.com/maximecb/gym-minigrid
+See https://github.com/Farama-Foundation/Minigrid.
 A simple, lightweight and fast Gym environments implementation of the famous gridworld.
 
-|  RL Algo | Empty | FourRooms | DoorKey | MultiRoom | Fetch |
-|----------|-------|-----------|---------|-----------|-------|
-| A2C      | | | | | |
-| PPO      | |  |  | | |
-| DDPG     | | | | | |
-| SAC      | | | | | |
-| TRPO     | | | | | |
+| RL Algo | Empty-Random-5x5   | FourRooms          | DoorKey-5x5        | MultiRoom-N4-S5    | Fetch-5x5-N2       | GoToDoor-5x5       | PutNear-6x6-N2     | RedBlueDoors-6x6   | LockedRoom         | KeyCorridorS3R1    | Unlock             | ObstructedMaze-2Dlh |
+| ------- | ------------------ | ------------------ | ------------------ | ------------------ | ------------------ | ------------------ | ------------------ | ------------------ | ------------------ | ------------------ | ------------------ | ------------------- |
+| A2C     |                    |                    |                    |                    |                    |                    |                    |                    |                    |                    |                    |                     |
+| PPO     | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark:  |
+| DDPG    |                    |                    |                    |                    |                    |                    |                    |                    |                    |                    |                    |                     |
+| SAC     |                    |                    |                    |                    |                    |                    |                    |                    |                    |                    |                    |                     |
```

Comment:
does it make sense for DDPG/SAC/TD3? I thought those env have discrete actions only?

Reply:
Indeed, I replaced SAC and DDPG by DQN and QR-DQN